test/libsolidity/lsp/references/import_aliases.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
Test names could be more specific. E.g.:
- `import_aliases.sol` -> `import_alias_for_module.sol`
- `import_aliases_2.sol` `import_aliases_for_imported_symbols.sol`